socorro/external/rabbitmq/crashstorage.py

```diff
@@ -315,11 +315,11 @@ class ReprocessingOneRabbitMQCrashStore(ReprocessingRabbitMQCrashStore):
     )
 
     def __init__(self, config, quit_check_callback=None):
-        super(ReprocessingRabbitMQCrashStore, self).__init__(
+        super(ReprocessingOneRabbitMQCrashStore, self).__init__(
             config, quit_check_callback=quit_check_callback
         )
```

Comment:
Unless I'm missing something, you can actually delete this ``__init__`` method, it does nothing the default doesn't do.